### ISSUE TITLE
Ferdi reloads when suspending the system

### ISSUE THREAD

**Issue by n0emis** [getferdi/ferdi#442]
**Describe the bug**
After waking my system from suspend, Ferdi reloads all services.
It always looks the same like when I restart ferdi, because it also connects to the API. 
Therefor I suspect that ferdi crashes but then restarts itself.

**Expected behavior**
Ferdi should stay open, even when suspending my laptop.

**Desktop:**
- OS: Arch Linux
- Kernel: 5.5.7-arch1-1
- Ferdi Version: 5.4.3
- Server Used: https://api.getferdi.com

**Additional information**
This is the console output I am getting during this:
```
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: AppStore.js:1:6788
Remote event names: suspend
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: AppStore.js:1:6909
Remote event names: resume
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: AppStore.js:1:5366
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: SettingsStore.js:1:2900
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: AppStore.js:1:5366
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: SettingsStore.js:1:3103
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: AppStore.js:1:5366
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: SettingsStore.js:1:2900
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: AppStore.js:1:5366
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: SettingsStore.js:1:3103
```

**Comment by kytwb** [comment]
Thank you for reporting this @n0emis!

**Comment by vantezzen** [comment]
Bug is probably introduced by

https://github.com/getferdi/ferdi/blob/eff719b87c60097342d393922048662c32255d88/src/stores/AppStore.js#L202-L212

**Comment by n0emis** [comment]
I tried update ferdi to the newest dev-version (5.4.4-beta.2 ), but this unfortunately didn't resolve my problem. But I was able to export the debug-infos:
https://debug.getferdi.com/6f7575ed-97a4-4cd7-a46b-7c74f229814b

**Comment by vantezzen** [comment]
Is you problem just that Ferdi restarts when you power your system back on? Because that is expected behaviour due to the code I sent above.

**Comment by n0emis** [comment]
> Is you problem just that Ferdi restarts when you power your system back on? Because that is expected behaviour due to the code I sent above.

@vantezzen Ah ok, sorry. I've overlooked your reply... Would it be possible to make this feature optional (deactivatable) with a config-setting?

**Comment by kytwb** [comment]
Shipped in [v5.4.4-beta.3](https://github.com/getferdi/ferdi/releases/tag/v5.4.4-beta.3).
